extensions/ql-vscode/src/view/model-editor/ModelOutputDropdown.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from "react";
+import { ChangeEvent, useCallback, useMemo } from "react";
+import { Dropdown } from "../common/Dropdown";
+import { ModeledMethod } from "../../model-editor/modeled-method";
+import { Method, getArgumentsList } from "../../model-editor/method";
+
+type Props = {
+  method: Method;
+  modeledMethod: ModeledMethod | undefined;
+  onChange: (method: Method, modeledMethod: ModeledMethod) => void;
+};
+
+export const ModelOutputDropdown = ({
+  method,
+  modeledMethod,
+  onChange,
+}: Props): JSX.Element => {
+  const argumentsList = useMemo(
```

Comment:
`argumentList` is only used to compute `options`, so we could merge these together and call `geArgumentList` on line 27 and it wouldn't change any behaviour. What do you think?

Reply:
Ah good point. I could do the same in `ModelInputDropdown`.

If we keep things as is we would be consistent with `ModelInputDropdown` and `ModelTypeDropdown` though.

I don't have strong opinions.